runtime/src/lib.rs

```diff
@@ -135,12 +137,18 @@ pub const MILLISECS_PER_BLOCK: u64 = 60_000;
 // NOTE: Currently it is not possible to change the slot duration after the chain has started.
 //       Attempting to do so will brick block production.
 pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
-pub const EPOCH_DURATION_IN_SLOTS: BlockNumber = 4 * HOURS;
+// pub const EPOCH_DURATION_IN_SLOTS: BlockNumber = 4 * HOURS;
+// pub const ERA_DURATION_IN_EPOCH: u32 = 6;
+
+pub const EPOCH_DURATION_IN_SLOTS: BlockNumber = MINUTES;
+pub const ERA_DURATION_IN_EPOCH: u32 = 4;
 
 // Time is measured by number of blocks.
 pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
+// Assumse month contains 30 days
+pub const MONTHS: BlockNumber = DAYS * 30;
 
 /// Existential deposit.
 pub const EXISTENTIAL_DEPOSIT: u128 = 500;
```

Comment:
This and all other network specific params should come from genesis config file. right?

Reply:
No, all this constants must be defined before build

Reply:
Understood. I was thinking about this and and will be great if we can have one config yaml file with all configurable values and use dev builds by default to use it with the idea to override it for testnet and mainnet. Possible?

Reply:
Definitely possible we have 2 options:
1. Throw chain spec file, but in this case we need to modify substrate pallets
2. Use some config file, but read it values while building project

Reply:
I think we should do the following:
1. Have default values in code for building.
2. Override them if config file is provided for node at runtime.
3. Provide a dev/test config file for testing and run the tests with the config file.

Reply:
I think it should be in a separate pr